docs/core/deploying/publish.md

```diff
@@ -0,0 +1,173 @@
+
+Starting with NET Core 2.1 SDK (version 2.1.300), .NET Core supports *patch version roll-forward*. When you create a self-contained deployment, .NET Core tools automatically include the latest serviced runtime of the .NET Core version that your application targets. (The latest serviced runtime includes security patches and other bug fixes.) The serviced runtime does not have to be present on your build system; it is downloaded automatically from NuGet.org. For more information, including instructions on how to opt out of patch version roll forward, see [Self-contained deployment runtime roll forward](runtime-patch-selection.md).
+
+FDD and SCD deployments use separate host executables, so you can sign a host executable for an SCD with your publisher signature.
+
+### Why deploy a self-contained deployment?
+
+Deploying a Self-contained deployment has two major advantages:
+
+- You have sole control of the version of .NET Core that is deployed with your app. .NET Core can be serviced only by you.
+
+- You're providing the version of .NET Core that it will run on, so you can be assured that the target system can run your .NET Core app.
+
+It also has a number of disadvantages:
+
+- Because .NET Core is included in your deployment package, you must select the target platforms for which you build deployment packages in advance.
+
+- You need to include .NET Core, your app, and its third-party dependencies, so the size of your deployment package is relatively large.
+
+  Size can also be mitigated by [Trimming](Trimming) (see section on Trimming later in this guide).
+
+- Deploying many self-contained .NET Core apps to a system can consume significant amounts of disk space, since each app duplicates .NET Core files.
+
+Self-contained deployment is available for Windows Client, Windows Server, MacOS, and Linux platforms.  
+
+To create a self-contained deployment for a Windows 10 x64 operating system, you would use the following command line:
+
+```dotnetcli
+dotnet publish -c Release -r win10-x64 --self-contained true
+```
+
+_Note: creating a self-contained deployment _**requires**_ that you specify a Runtime Identifier (RID) via the -r switch. _
+
+For a comprehensive list of all supported Runtime Identifiers (RIDs), refer to the [.NET Core RID Catalog](../rid-catalog.md).
+
+Note: Starting with .NET Core 2.0, you can reduce the size of your deployment on Linux systems by approximately 28 MB by using .NET Core globalization invariant mode. Ordinarily, .NET Core on Linux relies on the ICU libraries for globalization support. In invariant mode, the libraries are excluded with your deployment, and all cultures behave like the invariant culture.
```

Comment:
Same as earlier - just in case someone using 1.1 reaches the doc it doesn't hurt to say something only works 2.0+. I dont see a downside, only greater clarity.